client/src/pages/Tour.js

```diff
@@ -24,16 +33,9 @@ const Tour = () => {
       </p>
       <Standort>
         <Link to="/location">
-          <li>Standort</li>
-        </Link>
-        <Link to="/location">
-          <li>Standort</li>
-        </Link>
-        <Link to="/location">
-          <li>Standort</li>
-        </Link>
-        <Link to="/location">
-          <li>Standort</li>
+          {/* {list.items.map((item) => (
+            <li key={item}>{item}</li>
+          ))} */}
```

Comment:
Is this comment necessary?

Reply:
it just works as a reminder for myself, as the page isn't finished yet